vyper/semantics/analysis/module.py

```diff
@@ -907,6 +908,7 @@ def _import_to_path(level: int, module_str: str) -> PurePath:
 BUILTIN_PREFIXES = ["ethereum.ercs"]
 
 
+# TODO: could move this to analysis/common.py or something
```

Comment:
Sure why not?

Reply:
arguments against: analysis/common.py has the base visitor implementation (actually that should probably move to analysis/base.py but i digress). analysis/utils.py might be an ok place for it, but analysis/utils.py is already a grab bag (and causes import cycles!) so i'm not sure i want to put more stuff there.